site/gdocs/components/Recirc.scss

```diff
@@ -0,0 +1,85 @@
+.recirc {
+    --thumbnail-width: 70px;
```

Comment:
Technically this is meant to be one column wide, but writing this component with grids "all the way down" would have been a pain, so I'm check with Marwa to see if this is an okay compromise.